package.json

```diff
@@ -191,8 +194,8 @@
     "execa": "^2.0.4",
     "form-data": "^2.5.1",
     "hat": "0.0.3",
-    "interface-ipfs-core": "^0.110.0",
-    "ipfsd-ctl": "^0.44.1",
+    "interface-ipfs-core": "^0.111.0",
+    "ipfsd-ctl": "ipfs/js-ipfsd-ctl#fix/pubsub-flag-defaults",
```

Comment:
@alanshaw @hugomrdias we have a circular dependency issue here with `ipfsd-ctl`.

Reply:
Why not just have `--enable-pubsub` as the official option name and create an [alias](https://yargs.js.org/docs/#api-aliaskey-alias) as `--enable-pubsub-experiment`?

Reply:
So, are you recommending to `alias` here, and use in the tests `--enable-pubsub-experiment` for now?

Reply:
Anyway, I need the ctl to do: https://github.com/ipfs/js-ipfsd-ctl/pull/363/files#diff-d06a3c390787c7b35d2ab649265df1bbR50

Reply:
should i merge/release that PR as is ?

Reply:
If we do, we cannot delete this branch. But it is the only way to get CI green. My main concern is that we need to wait for the `js-ipfs` release 😡

Reply:
we can point to master or a commit until the release

Reply:
I am ok with merging that, and take the AI of PR ctl to use master once this PR is merged

Reply:
cool

Reply:
ctl released as 0.45.0